### PR TITLE
Laravel 9.x Compatibility

### PR DESCRIPTION
**Pull request by laravel-shift**
This is an automated pull request from [Shift](https://laravelshift.com) to update your package code and dependencies to be compatible with Laravel 9.x.

**Before merging**, you need to:

- Checkout the `l9-compatibility` branch
- Review **all** comments for additional changes
- Thoroughly test your package

If you do find an issue, please report it by commenting on this PR to help improve future automation.

### CODE REVIEW

**Comment by laravel-shift** [comment]
:warning: The latest version of Laravel has a dependency for `phpunit/phpunit` of `^9.3`. If you would like to also upgrade your tests, you may run the [PHPUnit 9 Shift](https://laravelshift.com/upgrade-phpunit-9) for free.

**Comment by laravel-shift** [comment]
:warning: Shift detected GitHub Actions which run jobs using a version `matrix`. Shift attempted to [update your configuration](https://github.com/facade/flare-client-php/pull/31/commits/841d0c7cfaeb1a98722f5575b6f331cd0876dd11?w=1) for Laravel 9. However, you should review these changes to ensure the desired combination of versions are built for your package.

**Comment by laravel-shift** [comment]
:alembic: **Using this package?** If you would like to help test these changes or believe them to be compatible, you may update your project to reference this branch.

To do so, temporarily add Shift's fork to the `repositories` property of your `composer.json`:

```json
{
    "repositories": [
        {
            "type": "vcs",
            "url": "https://github.com/laravel-shift/flare-client-php.git"
        }
    ]
}
```

Then update your dependency constraint to reference this branch:

```json
{
    "require": {
        "facade/flare-client-php": "dev-l9-compatibility",
    }
}
```

Finally, run: `composer update`

File: composer.json [hunk 1]
```diff
@@ -12,15 +12,15 @@
     "require": {
         "php": "^7.1|^8.0",
         "facade/ignition-contracts": "~1.0",
-        "illuminate/pipeline": "^5.5|^6.0|^7.0|^8.0",
+        "illuminate/pipeline": "^5.5|^6.0|^7.0|^8.0|^9.0",
         "symfony/http-foundation": "^3.3|^4.1|^5.0",
         "symfony/mime": "^3.4|^4.0|^5.1",
         "symfony/var-dumper": "^3.4|^4.0|^5.0"
```
Comment (snoek09) [inline composer.json:18]: This needs to be updated to support version ^6.0 of symfony/var-dumper as Laravel 9 requires this version.

**Pull request closed without merging**